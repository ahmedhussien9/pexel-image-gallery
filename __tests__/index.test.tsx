import { render, screen } from '@testing-library/react'
import IndexPage from '../pages/index'

describe('Home', () => {
    it('renders Photo gallery page', () => {
        const { container } = render(<IndexPage photos={null} />)
        expect(container).toMatchSnapshot()
    });
    
})