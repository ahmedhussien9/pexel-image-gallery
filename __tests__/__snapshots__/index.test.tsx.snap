// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders Photo gallery page 1`] = `
<div>
  <div>
    <header>
      <nav
        class="nav"
      >
        <ul
          class="list container"
        >
          <li
            class="item"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-biking logo"
              data-icon="biking"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 640 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M400 96a48 48 0 1 0-48-48 48 48 0 0 0 48 48zm-4 121a31.9 31.9 0 0 0 20 7h64a32 32 0 0 0 0-64h-52.78L356 103a31.94 31.94 0 0 0-40.81.68l-112 96a32 32 0 0 0 3.08 50.92L288 305.12V416a32 32 0 0 0 64 0V288a32 32 0 0 0-14.25-26.62l-41.36-27.57 58.25-49.92zm116 39a128 128 0 1 0 128 128 128 128 0 0 0-128-128zm0 192a64 64 0 1 1 64-64 64 64 0 0 1-64 64zM128 256a128 128 0 1 0 128 128 128 128 0 0 0-128-128zm0 192a64 64 0 1 1 64-64 64 64 0 0 1-64 64z"
                fill="currentColor"
              />
            </svg>
          </li>
          <li
            class="item link"
          >
            <label
              class="switch"
            >
              <input
                type="checkbox"
              />
              <span
                class="slider"
              />
            </label>
          </li>
        </ul>
      </nav>
    </header>
    <div
      class="container"
    >
      <div
        class="PageHeader"
      >
        <h1>
          Image Gallery
        </h1>
        <p>
          This is a description for the image gallery page
        </p>
      </div>
      <div
        class="list"
      />
    </div>
    <footer
      class="Footer"
    >
      <span>
        I'm here to stay (Footer)
      </span>
    </footer>
  </div>
</div>
`;
